tests/tier2/tc_2065_deprecation_warning_for_etc_sysconfig.py

```diff
@@ -0,0 +1,49 @@
+# coding:utf-8
+from virt_who import *
+from virt_who.base import Base
+from virt_who.register import Register
+from virt_who.testing import Testing
+
+class Testcase(Testing):
+    def test_run(self):
+        self.vw_case_info(os.path.basename(__file__), case_id='RHEL-195890')
+        self.vw_case_init()
+
+        # case config
+        results = dict()
+        sysconf_file = "/etc/sysconfig/virt-who"
```

Comment:
need to think about  rhel9, /etc/sysconfig/virt-who is not exist in rhel9

Reply:
Skipped test case for RHEL9.